### PR TITLE
Deployment workflow changes

### PR DESCRIPTION
**Pull request by mafonso**
Changes to deployment workflow for Engine

### CODE REVIEW

File: bin/deploy [hunk 1]
```diff
@@ -15,9 +15,9 @@ cf login -a $CF_ENDPOINT -u $CF_USER -p "$CF_PASSWORD" -o $CF_ORG -s $CF_SPACE
 cf a
 
 # Notify Slack deployment started
-curl -X POST \
---data-urlencode 'payload={"text": "'"Deploying $CF_APP.\n${GIT_MESSAGE}"'", "channel": "'"#$SLACK_CHANNEL"'", "username": "cf-deploy", "icon_emoji": ":cloud:", "fields": [{"title": "ENV", "value": "'"$CF_SPACE"'", "short": true}, {"title": "SHA1", "value": "'"<https://github.com/bitzesty/trade-tariff-frontend/compare/$GIT_OLD_REVISION...$GIT_REVISION|$GIT_REVISION>"'", "short": true}]}' \
-$SLACK_WEBHOOK
+#curl -X POST \
+#--data-urlencode 'payload={"text": "'"Deploying $CF_APP.\n${GIT_MESSAGE}"'", "channel": "'"#$SLACK_CHANNEL"'", "username": "cf-deploy", "icon_emoji": ":cloud:", "fields": [{"title": "ENV", "value": "'"$CF_SPACE"'", "short": true}, {"title": "SHA1", "value": "'"<https://github.com/bitzesty/trade-tariff-frontend/compare/$GIT_OLD_REVISION...$GIT_REVISION|$GIT_REVISION>"'", "short": true}]}' \
+#$SLACK_WEBHOOK
```
Comment (tavy88) [inline bin/deploy:20]: Can we not use our Engine Slack here instead? Any blockers?

File: bin/deploy [hunk 2]
```diff
@@ -44,6 +44,6 @@ cf add-network-policy $CF_APP --destination-app $PRIVATE_APP --protocol tcp --po
 cf attach-autoscaling-policy $CF_APP config/autoscaling/"$CF_SPACE"-policy.json
 
 # Notify Slack deployment finished
-curl -X POST \
---data-urlencode 'payload={"text": "'"Deployment finished for $CF_APP"'", "channel": "'"#$SLACK_CHANNEL"'", "username": "cf-deploy", "icon_emoji": ":cloud:", "fields": [{"title": "ENV", "value": "'"$CF_SPACE"'", "short": true}, {"title": "SHA1", "value": "'"<https://github.com/bitzesty/trade-tariff-frontend/compare/$GIT_OLD_REVISION...$GIT_REVISION|$GIT_REVISION>"'", "short": true}]}' \
-$SLACK_WEBHOOK
+#curl -X POST \
+#--data-urlencode 'payload={"text": "'"Deployment finished for $CF_APP"'", "channel": "'"#$SLACK_CHANNEL"'", "username": "cf-deploy", "icon_emoji": ":cloud:", "fields": [{"title": "ENV", "value": "'"$CF_SPACE"'", "short": true}, {"title": "SHA1", "value": "'"<https://github.com/bitzesty/trade-tariff-frontend/compare/$GIT_OLD_REVISION...$GIT_REVISION|$GIT_REVISION>"'", "short": true}]}' \
+#$SLACK_WEBHOOK
```
Comment (tavy88) [inline bin/deploy:49]: Can we not use our Engine Slack here instead? Any blockers?

File: bin/deploy (hunk 1 above)
Comment (mafonso) [reply bin/deploy:20]: Yes, we can and we should.  I don't have workspace permissions to create this webhook.   
But once we have our own webhook I'll  update it.

Same goes for config for sentry and scoutapm

Comment (tavy88) [reply bin/deploy:20]: Fair!

**Review by tavy88 — APPROVED** [review]

**Pull request merged**